docs/source/user_guide/timeseries.ipynb

```diff
@@ -314,18 +321,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "res = pdc.get_trend_dataframe(X_train_time, y_train_time)\n",
-    "fig, axs = plt.subplots(4)\n",
-    "fig.set_size_inches(18.5, 14.5)\n",
-    "axs[0].plot(y, \"r\")\n",
-    "axs[0].set_title(\"signal\")\n",
-    "axs[1].plot(res[0][\"trend\"], \"b\")\n",
-    "axs[1].set_title(\"trend\")\n",
-    "axs[2].plot(res[0][\"seasonality\"], \"g\")\n",
-    "axs[2].set_title(\"seasonality\")\n",
-    "axs[3].plot(res[0][\"residual\"], \"y\")\n",
-    "axs[3].set_title(\"residual\")\n",
-    "plt.show()"
+    "res = pdc.plot_decomposition(X_train_time, y_train_time)"
```

Comment:
the image isn't showing [here](https://feature-labs-inc-evalml--3835.com.readthedocs.build/en/3835/user_guide/timeseries.html#Polynomial-Decomposer) for some reason - can you double check?

Reply:
I'm seeing it with no problem - are you sure it's missing?

Reply:
I can see it now - might have just been my browser!